### ISSUE TITLE
Use more succinct audio cue when terminal command fails

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#175012]
We should use `error on line` instead of `task failed`

cc @jooyoungseo

**Comment by eleanorjboyd** [comment]
What would be an example of a task to try? I tried a build task with a compile failure but that has a really long error message.

**Comment by meganrogge** [comment]
type "failure" into a terminal with screen reader mode on. you'll hear a sound that is much shorter than the task failure one
